mantidproject/mantid
# sample logs dialog: hide plot if no suitable logs in the workspace

**Description of work.**
See commit messages

**Report to:** @gvardany 


**To test:**
Data setup script
``` Python
no_ts_logs = Load('ILL/D22/192068.nxs')
ts_logs = Load("ENGINX00228061")
mdws = CreateMDWorkspace(Dimensions=1, Extents='-1,1', Names='A', Units='U')
mdws.addExperimentInfo(ts_logs)
mdws.addExperimentInfo(no_ts_logs)
```
1. Open workbench and run the script, you will get 3 workspaces
1. ts_logs -> show sample logs
   - You should see the plot or statistics boxes 
   - plots should work etc.
1. no_ts_logs -> show sample logs
   - You should not see the plot or statistics boxes
1. mdws-> show sample logs
   - The plot and stats should appear and be removed as you change the experiment Info spinbox
   - plotting etc should still work sensibly


re #28569



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
